src/mixins/legal-api-mixin.ts

```diff
@@ -254,4 +254,59 @@ export default class LegalApiMixin extends Mixins(CommonMixin) {
       })
     })
   }
+
+  /** Business Digital Credential Helpers **/
+
+  /**
+   * Fetches digital credential information.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns a promise to return the credentials connection data
+   */
+  async fetchCredentials (businessId: string): Promise<any> {
+    const url = `businesses/${businessId}/digitalCredentials`
+    return axios.get(url)
+      .catch(error => {
+        console.log(error.message)
+      })
+  }
+
+  /**
+   * Creates a digital credentials invitation.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns a promise to return the invitation data
+   */
+  async createCredentialInvitation (businessId: string): Promise<any> {
+    const url = `businesses/${businessId}/digitalCredentials/invitation`
+    return axios.post(url)
+      .catch(error => {
+        console.log(error.message)
+      })
+  }
+
+  /**
+   * Fetches a digital credentials connection information.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns a promise to return the credentials connection data
+   */
+  async fetchCredentialConnection (businessId: string): Promise<any> {
+    const url = `businesses/${businessId}/digitalCredentials/connection`
+    return axios.get(url)
+      .catch(error => {
+        console.log(error.message)
+      })
+  }
+
+  /**
+   * Issue a digital credentials offer.
+   * @param businessId The business identifier (aka entity inc no)
+   * @param credentialType The credential offer type
+   * @returns a promise to return the credentials connection data
+   */
+  async issueCredentialOffer (businessId: string, credentialType: DigitalCredentialTypes): Promise<any> {
+    const url = `businesses/${businessId}/digitalCredentials/${credentialType}`
+    return axios.post(url)
+      .catch(error => {
+        console.log(error.message)
+      })
```

Comment:
These all look good.

Do the callers handle null/undefined return values in case the network calls fails? (I ask because the catch blocks absorb the error and don't return anything.)

Reply:
Definitely intended, due to the nature of these requests. 
The callers do handle these but there isn't any prompts with regards to errors because 404s etc are expected. 

The jist, the UI tries to keep in sync with whats going on between api and users mobile wallet by looking up data between steps.